cardano-api/src/Cardano/Api/Fees.hs

```diff
@@ -488,15 +489,14 @@ evaluateTransactionExecutionUnits _eraInMode systemstart history pparams utxo tx
                          (Map ScriptWitnessIndex
                               (Either ScriptExecutionError ExecutionUnits))
     evalAlonzo era tx =
-      case Alonzo.evaluateTransactionExecutionUnits
+      fromLedgerScriptExUnitsMap . fromRight mempty <$>
```

Comment:
We should at the very least leave a `TODO` comment saying that this `fromRight mempty` is hiding a basic validation error in `Alonzo.evaluateTransactionExecutionUnits`. This will be very confusing to anyone hitting the `Left` case here!